openvm/src/plonk/bus_interaction_handler.rs

```diff
@@ -56,11 +57,15 @@ pub fn add_bus_to_plonk_circuit<T>(
         .args
         .iter()
         .chain([bus_interaction.mult].iter())
-        .zip(
-            gates
-                .iter_mut()
-                .flat_map(|gate| [&mut gate.a, &mut gate.b, &mut gate.c]),
-        )
+        .zip(gates.iter_mut().flat_map(|gate| {
```

Comment:
```suggestion
        .zip_eq(gates.iter_mut().flat_map(|gate| {
```

Are they expected to be the same length?